openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -175,6 +175,22 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                       </div>
                   </div>
                   {% endif %}
+                  {% if interface.wireless.bitrate %}
+                  <div class="form-row">
+                      <label>{% trans 'Bitrate' %}:</label>
+                      <div class="readonly">
+                          {{ interface.wireless.bitrate }} KBits/s
```

Comment:
can we do some preprocessing to show MBit/s which is probably more user friendly?